### PR TITLE
NEPT-1094: Define the component social_icon.

### PR DESCRIPTION
**Pull request by voidtek**
### Description

Define the component social_icon into nexteuropa_formatters

### Change log

- Added: the definition of the component social_icon.

### Commands

```sh
no commands

```

### CODE REVIEW

File: profiles/common/modules/custom/nexteuropa_formatters/nexteuropa_formatters_fields/nexteuropa_formatters_fields.module [hunk 1]
```diff
@@ -57,6 +57,26 @@ function nexteuropa_formatters_fields_field_formatter_info() {
...
       ),
     ),
+    'ne_social_icon' => array(
+      'label' => t('NE Social Icon'),
+      'description' => t('Displays as a Social Icon'),
+      'field types' => array(
+        'link_field',
+      ),
+      'settings' => array(
+        'type' => 'email',
```
Comment (Fefaine) [inline profiles/common/modules/custom/nexteuropa_formatters/nexteuropa_formatters_fields/nexteuropa_formatters_fields.module]: I don't understand why you are using an email type?

Comment (drupol) [reply profiles/common/modules/custom/nexteuropa_formatters/nexteuropa_formatters_fields/nexteuropa_formatters_fields.module]: This is the formatter's default config.

Comment (drupol) [reply profiles/common/modules/custom/nexteuropa_formatters/nexteuropa_formatters_fields/nexteuropa_formatters_fields.module]: See: https://api.drupal.org/api/drupal/modules%21field%21field.api.php/function/hook_field_formatter_info/7.x

Comment (voidtek) [reply profiles/common/modules/custom/nexteuropa_formatters/nexteuropa_formatters_fields/nexteuropa_formatters_fields.module]: I confirm, it's the default value for this type of component.
I don't want put a social network (like facebook, twitter or other) as a default, so I choose one value that exists and that every body knows. 
We can try to put with no default value.

Comment (drupol) [reply profiles/common/modules/custom/nexteuropa_formatters/nexteuropa_formatters_fields/nexteuropa_formatters_fields.module]: 👍

**Comment by netlooker** [comment]
👍

**Pull request merged**